vector/src/main/java/im/vector/activity/VectorRoomActivity.java

```diff
@@ -3855,72 +3855,74 @@ public void onUnexpectedError(Exception e) {
      * Assume he wants to update it.
      */
     private void onRoomTitleClick() {
-        LayoutInflater inflater = LayoutInflater.from(this);
+        if (null!= mRoom && !RoomUtils.isDirectChat(mSession,mRoom.getRoomId())) {
```

Comment:
We should disable the listeners creation instead of disabling the action on click